pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -72,6 +72,22 @@ type ResourceBindingSpec struct {
 	// +optional
 	Clusters []TargetCluster `json:"clusters,omitempty"`
 
+	// GracefulEvictionTasks holds the eviction tasks that are expected to perform
+	// the eviction in a graceful way.
+	// The intended workflow is:
+	// 1. Once the controller(such as 'taint-manager') decided to evict the resource that
+	//    is referenced by current ResourceBinding or ClusterResourceBinding from a target
+	//    cluster, it removes(or scale down the replicas) the target from Clusters(.spec.Clusters)
```

Comment:
> (or scale down the replicas)

Do we need to move the scale down replicas into `GracefulEvictionTasks`?

Reply:
Yes, e.g.
```yaml
spec:
  clusters:
  - name: member2
    replicas: 10
  - name: member1
    replicas: 10
```
evict `5` replicas from `member1`, then
```yaml
spec:
  clusters:
  - name: member2
    replicas: 10
  - name: member1
    replicas: 5
  gracefulEvictionTasks:
  - name: member1
    replicas: 5
    ...
```

Reply:
This may cause problems. `.spec.clusters` describes the current cluster replicas distribution status. When the number of  member2 replicas changes from 10 to 5, the number of workload replicas will be directly changed when `ensuringWork` in binding-controller. This will conflict with the eviction behavior.

Reply:
> When the number of member2 replicas changes from 10 to 5,

`member2` do you mean `member1`?

Reply:
Clerical error. It does refer to `member1`.

Reply:
Yeah, that would be too complicated, and given the real use case is uncertain yet, I suppose we can remove the `replicas` from `gracefulEvictionTasks`. Now only focus on evicting the entire resource from clusters. 

@Garrybest What do you say?